confidence.py

```diff
@@ -285,7 +344,9 @@ def read_envvar_file(name):
 
 # ordered sequence of name templates to load, in increasing significance
 LOAD_ORDER = (
+    read_xdg_config_dirs,
     '/etc/{name}.{extension}',
+    read_xdg_config_home,
```

Comment:
I tested XDG on windows, it will always result in None. The current load order does not support windows machines when doing a load by name and having a file outside the project root

Reply:
workaround is placing the required config fpath as an ENV var

Reply:
Took some effort to test this on Windows, but the config home will work on Windows; the expansion of `~/.config` works and reading from `~/.config/example.yaml` works as expected. The caveat is that creating a directory named `.config` needs to be done outside `explorer.exe`, so this isn't very ideal. Let me add a new issue to support the more 'default' locations on Windows (and maybe OSX too? `LOAD_ORDER` is gonna get long :cry:).